composer.json

```diff
@@ -57,6 +57,9 @@
         }
     },
     "config": {
-        "preferred-install": "dist"
+        "preferred-install": "dist",
+        "allow-plugins": {
+            "bamarni/composer-bin-plugin": true
```

Comment:
Not explicitly allowing plugins is deprecated and will be removed in July 2022.
https://getcomposer.org/doc/06-config.md#allow-plugins